README.md

```diff
@@ -176,23 +176,120 @@ You can install the package via Composer:
 composer require spatie/laravel-permission
 ```
 
-Copy `vendor/spatie/laravel-permission/config/permission.php` to `config/permission.php`
+Copy the required files:
 
-Copy `vendor/spatie/laravel-permission/database/migrations/create_permission_tables.php.stub` to `database/migrations/2017_12_01_010101_create_permission_tables.php`
+```bash
+cp vendor/spatie/laravel-permission/config/permission.php config/permission.php
+cp vendor/spatie/laravel-permission/database/migrations/create_permission_tables.php.stub database/migrations/create_permission_tables.php
+```
```

Comment:
The migration MUST have the date prefix on the filename, else it will trigger an error.

Reply:
Fixed. :)